ui/libexec/dockerutils.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
+
+
+# Helpers for spammy sections
+img_build_fail() {
+    error "Failed to build the installer image. Check the log for details."
+    die "You may need to run '$0 --clean' to clean up Docker"
+}
+
+img_pull_fail() {
+    die "Failed to pull the installer image. Check the log for details."
+    die "You may need to run '$0 --clean' to clean up Docker"
+}
+
+ui_tarball() {
```

Comment:
Permissions should be set in git and carried through into the container during docker COPY command.  I was testing based on IntelliJ uploading changes rather than git pull, and that's why this is only showing up when the installer container is built from git cloned sources.